src/global.type.ts

```diff
@@ -42,6 +43,15 @@ export interface JRRun {
   reports: JRReport[];
 }
 
+// A run is composed of multiple junit files
+export interface JRCase {
+  id: string;
+  name: string;
+  suite: string;
+  duration: number;
+  statis: string;
```

Comment:
```suggestion
  state: string;
```

Reply:
according to its usage in `const zcPayload: ZenCrepesStateNode`
Also, aren't we missing these: jahia, module, caseTotal, caseSuccess, caseFailure and createdAt?

Reply:
thanks, I added the missing ones and corrected the typo